### DISCUSSION TITLE
Xdrip WhatsApp emergency

### DISCUSSION THREAD

**Discussion by hasanjaved25** [NightscoutFoundation/xDrip#1906] [Ideas]
I have a feature request.
There is an awesome option in xdrip that we can send emergency SMS msgs to selected contacts. I hope we can have a similar option to send WhatsApp msgs in xdrip. Is it possible?

**Comment by MasterPlexus** [comment]
Unfortunately WhatsApp did not provide an useable API without interchange of the user. So in this case all what can be done is to let the app open with a predefined text, but the user has to push the selection and the Sendbutton manually ... So no unhackable way for automatism...
But anyhow: I did not see the better use case for Whatsapp or a other messenger, as SMS is an guaranteed technic wich works on all mobile's safely ..
